contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -504,6 +505,11 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     bool nativePayment,
     bool onlyPremium
   ) internal view returns (uint96) {
+    // fulfilRandomWords msg.data has 772 bytes and with an additional
+    // buffer of 32 bytes, we get 804 bytes.
+    if (msg.data.length > 804) {
+      revert MsgDataTooBig(uint32(msg.data.length), 804);
```

Comment:
can this cast fail? thinking of a scenario where a front runner sets an extremely large calldata. Not a big threat because tx was going to revert anyways but will give a better revert reason. Should we just use uint256 for `have`?

Reply:
Sure, I can do that. I was concerned about contract bytecode size and gas usage. Does it have any impact?